### ISSUE TITLE
Add background-color CSS property

### ISSUE THREAD

**Issue by funnym0nk3y** [dendronhq/dendron#1937]
## Please select if your request is either something new or an enhancement ##

- [x] Enhancement of an existing Feature.
- [ ] Request of a new feature.

## Please select the area your request applies to. (Multiple selections are Possible. You can leave blank if you're not sure.) ##

- [ ] Workspace - VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup - Dendron's Lookup Command
- [x] Views - Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema - Dendron Schemas
- [ ] Pod - Data import from / export to Dendron
- [ ] Publish - External Site Publish
- [ ] Markdown - Markdown features, such as syntax support and features

Please add the CSS property `background-color` to the allowed porperties to avoid unreadable images with transparent background.

**Comment by hikchoi** [comment]
Thanks for the suggestion. We are tracking various requests for preview and customizable CSS has been requested a lot. Unfortunately this is not possible at the moment but we plan to support it in the near future.

**Comment by ScriptAutomate** [comment]
Related GitHub discussion:
- https://github.com/dendronhq/dendron/discussions/1298

**Comment by aleksey-rowan** [comment]
Hey, @funnym0nk3y, Dendron 0.97 has shipped with [partial theme support for note preview](https://wiki.dendron.so/notes/lb9wd7z62ch7b4slscp05i4/). Does this solve your issue?

**Comment by funnym0nk3y** [comment]
@aleksey-rowan Is it possible to only set a different background color for images?

**Comment by funnym0nk3y** [comment]
@aleksey-rowan It does solve my problem. But how can I change only that one property and not the whole default theme? It was using some of my vscode colors. I tried it with [packages/dendron-plugin-views/assets/css/main/dark.css](..packages/dendron-plugin-views/assets/css/main/dark.css), but could not get it to work.

**Comment by aleksey-rowan** [comment]
@funnym0nk3y, yes, you are right, this doesn't seem to be possible right now--as soon as you start using the custom theme, it applies the default light/dark theme. 

Could you please open another issue about the custom theme inheriting the current VSCode theme and not the built-in theme (or maybe an option to chose what the custom theme should be inheriting)?

**Comment by aleksey-rowan** [comment]
@funnym0nk3y, thanks for opening an issue on theme inheritance!

Since this issue seems to be done, it'll close in a couple of days unless you have further comments. Thank you!
